.github/workflows/argus-stack-staging-upsert.yaml

```diff
@@ -13,12 +10,22 @@ concurrency:
 jobs:
   run_if:
     if: github.head_ref == 'release-please--branches--main'
+    outputs:
+      changes: ${{ steps.filter.outputs.staging_config }}
     runs-on: [ARM64, self-hosted, Linux]
     steps:
       - run: echo "The head branch of this PR is the release please branch"
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            staging_config:
+              - '.infra/common.yaml'
+              - '.infra/staging/values.yaml'
```

Comment:
Make sure this always runs so we can make it a required check without blocking feature branch PR merges